hal-sim/hal_impl/mode_helpers.py

```diff
@@ -5,114 +5,114 @@
 from . import data
 hal_data = data.hal_data
 from . import functions as fns
-    
+
 def notify_new_ds_data():
     '''Called when driver station data is modified'''
     data.hooks.notifyDSData()
 
 def set_autonomous(enabled):
     '''Only designed to be called on transition'''
-    
+
     hal_data['control'].update({
         'autonomous': True,
         'test': False,
         'enabled': enabled,
         'dsAttached': True
     })
-    
+
     if enabled:
-        hal_data['time']['match_start'] = fns.hooks.getFPGATime()
+        hal_data['time']['match_start'] = 135000000
     else:
         hal_data['time']['match_start'] = None
-    
+
     notify_new_ds_data()
-    
+
 
 def set_test_mode(enabled):
     '''Only designed to be called on transition'''
-    
+
     hal_data['control'].update({
         'autonomous': False,
         'test': True,
         'enabled': enabled,
         'dsAttached': True
     })
-    
+
     hal_data['time']['match_start'] = None
     notify_new_ds_data()
-    
+
 
 def set_teleop_mode(enabled):
     '''Only designed to be called on transition'''
-    
+
     hal_data['control'].update({
         'autonomous': False,
         'test': False,
         'enabled': enabled,
         'dsAttached': True
     })
-    
+
     if enabled:
-        hal_data['time']['match_start'] = fns.hooks.getFPGATime() - 15000000
+        hal_data['time']['match_start'] = 135000000 - 15000000
```

Comment:
I think we should change the name of this key to `time_remaining` and currently, if I start the sim, wait 20 seconds, then enable again, the time won't be at the beginning of teleop, instead it'll say there are 100 seconds left. I don't think this is desired behavior. I think you should do `fns.hooks.getFPGATime() + 120000000`

Reply:
I feel like just calling it `remaining` may be preferable, given that `time` is used in the parent dictionary.